nyxtools/flyer-eiger2.py

```diff
@@ -16,37 +16,15 @@
 DEFAULT_DATUM_DICT = {"data": None, "omega": None}
 
 
-class NYXFlyer(MXFlyer):
+class NYXEiger2Flyer(MXFlyer):
     def __init__(self, vector, zebra, detector=None) -> None:
-        self.name = "NYXFlyer"
-        self.vector = vector
-        self.zebra = zebra
-        self.detector = detector
-
+        self.super().__init__(vector, zebra, detector)
+        self.name = "NYXEiger2Flyer"
         self.data_directory_name = None
```

Comment:
do you really need to keep this variable in?

Reply:
I was not sure, so I decided to leave it. We could remove it and see if any failures occur due to it being missing, but I our time-line was originally going to be very short so I thought getting it functional quickly would be more important.